Gemfile.lock

```diff
@@ -82,21 +82,21 @@ GEM
     ast (2.4.2)
     autoprefixer-rails (10.2.4.0)
       execjs
-    aws-eventstream (1.1.0)
-    aws-partitions (1.425.0)
-    aws-sdk-core (3.112.0)
+    aws-eventstream (1.1.1)
```

Comment:
### [aws-eventstream](https://github.com/aws/aws-sdk-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.0 → 1.1.1

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-eventstream/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-eventstream)_]





<details>
<summary>Commits</summary>

A change of **1** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.1.0...1.1.1).

These are the individual commits:
- (cfd13e3) [Version 1.1.1 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/cfd13e3718110808e3074d5a2f96c8d01815d52d)


</details>